### ISSUE TITLE
Deprecation warning with node 7.0.0.

### ISSUE THREAD

**Issue by ryanlewis** [archiverjs/node-archiver#204]
The following warning is returned by Node.

    (node:64756) DeprecationWarning: Using Buffer without `new` will soon stop working. Use `new Buffer()`, or preferably `Buffer.from()`, `Buffer.allocUnsafe()` or `Buffer.alloc()` instead.

Code that triggers this warning.

```js
  const fs = require('fs')
  const path = require('path')
  const archiver = require('archiver')

  const files = [ 'package.json' ]

  const zipFile = fs.createWriteStream('myzipfile.zip')
  const archive = archiver('zip')
  archive.pipe(zipFile)
  files.forEach(file => archive.append(fs.createReadStream(file), { name: path.basename(file) }))
  archive.finalize()
```

Node 7 is installed via NVM.

**Comment by asprouse** [comment]
This can be traced to using an outdated version of `buffer-crc32`. This project uses 0.2.1 and the warning was fixed in 0.2.7

**Comment by ctalkington** [comment]
the way the package version is defined you should be able to clear npm cache and get the latest 0.2.x release
